gradle.properties

```diff
@@ -1,4 +1,6 @@
 version=1.1.1-SNAPSHOT
+pelias_version=1.0.0
```

Comment:
This is cool. Should we do pull out all the dependency version numbers into props (not just Mapzen libs)?